server_admin/topics/authentication/password-policies.adoc

```diff
@@ -7,7 +7,12 @@ When {project_name} creates a realm, it does not associate password policies wit
 
 .Procedure
 . Click *Authentication* in the menu.
+ifeval::[{project_community}==true]
+. Click the *Policies* tab.
+endif::[]
+ifeval::[{project_product}==true]
 . Click the *Password Policy* tab.
+endif::[]
 . Select the policy to add in the *Add policy* drop-down box.
 . Enter a value for the *Policy Value* corresponding with the policy chosen.
```

Comment:
There is no *Policy Value* in the new UI.  Just a place where a value show up.  I think you can just eliminate the *Policy Value* phrase to work for both old and new UI.

. Enter a value that applies to the policy chosen.